beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -694,6 +700,20 @@ impl<T: BeaconChainTypes> std::convert::From<ReadyWork<T>> for WorkEvent<T> {
                     seen_timestamp,
                 },
             },
+            ReadyWork::LCUpdate(QueuedLCUpdate {
```

Comment:
I think it would be nice to use the full name here specially for the public structs and enums. 
`ReadyWork::LightClientUpdate`, `QueuedLightClientUpdate` etc